.npmignore

```diff
@@ -0,0 +1,4 @@
+.travis.yml
+History.md
+Readme.md
```

Comment:
This file is unconditionally included in npm package, let's remove it from here

Reply:
Sure, although I it may need to be renamed to `README.md`. Like me to take care fo that?

Reply:
That's good idea, thanks!